### ISSUE TITLE
does

### ISSUE THREAD

**Issue by ucasiggcas** [CrawlScript/tf_geometric#24]

**Comment by ucasiggcas** [comment]
hi,dear
for the rp, if I want to train much data ,could I run it quickly with tfg ?

**Comment by hujunxianligong** [comment]
Use @tf_utils.function to accelerate your code.

**Comment by hujunxianligong** [comment]
Do you use WeChat?

**Comment by ucasiggcas** [comment]
emm, could I use multi-gpu with the tfg ?

**Comment by hujunxianligong** [comment]
There are two demos that use multi-gpu:
- https://github.com/CrawlScript/tf_geometric/blob/master/demo/demo_distributed_mean_pool.py
- https://github.com/CrawlScript/tf_geometric/blob/master/demo/demo_distributed_gcn.py

**Comment by ucasiggcas** [comment]
大佬，有没有直接用tf来做多GPU训练的啊？
或者tfg与tf/keras 是一样的吗？
有没有推荐方面的应用啊？
多谢

**Comment by hujunxianligong** [comment]
tfg实现的就是tf.keras模型，你tf.keras平时怎么分布式怎么多GPU，这个就可以怎么用。

VideoRec ***@***.***> 于2021年4月11日周日 下午2:46写道：

**Comment by wxue1995** [comment]
GAT能使用在三维上吗？

**Comment by hujunxianligong** [comment]
请问三维指的是什么？类似点云的三维数据，还是加了时间维度的temporal gat？-------- 原始邮件 --------发件人： wxue1995 ***@***.***>日期： 2021年8月11日周三 上午9:13收件人： CrawlScript/tf_geometric ***@***.***>抄送： hujunxianligong ***@***.***>, Comment ***@***.***>主    题： Re: [CrawlScript/tf_geometric] does (#24)

GAT能使用在三维上吗？

**Comment by wxue1995** [comment]
3各维度类似于【batch，length，dim】，也就相当于一条语句有length长度个节点，每个节点维度为dim。Mini_batch思想，一次送入batch条长度为length长度的图节点语料。

从 Windows 版邮件发送

发件人: hujunxianligong
发送时间: 2021年8月12日 8:18
收件人: CrawlScript/tf_geometric
抄送: wxue1995; Comment
主题: Re: [CrawlScript/tf_geometric] does (#24)

请问三维指的是什么？类似点云的三维数据，还是加了时间维度的temporal gat？-------- 原始邮件 --------发件人： wxue1995 ***@***.***>日期： 2021年8月11日周三 上午9:13收件人： CrawlScript/tf_geometric ***@***.***>抄送： hujunxianligong ***@***.***>, Comment ***@***.***>主 题： Re: [CrawlScript/tf_geometric] does (#24) 
GAT能使用在三维上吗？ 

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by hujunxianligong** [comment]
你要的是tfg.BatchGraph的功能，天然支持的。
